src/QsCompiler/Tests.Compiler/RegexTests.fs

```diff
@@ -77,24 +77,20 @@ let ``Strip unique variable name resolution`` () =
             "'TName"
         ]
 
-    origNames
-    |> List.map (fun var -> var, UniqueVariableNames.StripUniqueName var)
-    |> List.iter Assert.Equal
+    for var in origNames do
```

Comment:
Liked this change! The code is easier to read now :)

Reply:
Yeah, the linter complained about repeated calls to List.map that could be combined, but I realized it was simpler to rewrite it as a for loop.